Hide list of panes for global dashboard panes

megoth opened the pull request:
This should solve https://github.com/solid/solid-panes/issues/142

Vinnl commented on outline/manager.js:
```diff
@@ -381,7 +381,7 @@ module.exports = function (doc) {
 
   function expandedHeaderTR (subject, requiredPane, options) {
-    function renderPaneIconTray (td) {
+    function renderPaneIconTray (td, hideList) {
```
To avoid making our API sprawl (`renderPaneIconTray(td, false, true, undefined, null, true)`) even worse, maybe make the second parameter an `options` object?

megoth replied:
I agree with the need to avoid API sprawl, but perhaps add an `options` object when we add more parameters? Feels a bit weird to do for one property.

Vinnl replied:
But that means breaking the API, which is especially painful in a file that is not written in TypeScript :) (i.e. you'd not only have to update the parameters here, but also everwhere this function is called - though I guess that might be only in this file?)

megoth replied:
Ah, right, that's a good argument. Ok, I'll make it into an option-parameter.

Vinnl reviewed (changes requested):
Ah, that works as well. Could you squash the commits? :)

megoth committed:
WIP: Not working tray removal in the dashboard

A bit hacky, but seems to work

This should solve https://github.com/solid/solid-panes/issues/142

megoth committed:
Making hideList into an options parameter
